### DISCUSSION TITLE
Made a simple diagram for the Smart Contract Lottery

### DISCUSSION THREAD

**Discussion by sergipastor** [Cyfrin/foundry-full-course-cu#2350] [Show and tell]
Hey guys, 
I've just finished the Foundry Fundamentals course and I decided to draw a simple diagram of the system to better understand the flow of info within the project.

![raffle_diagram](https://github.com/user-attachments/assets/2b1a87eb-878c-4e65-b5f9-fbc23f7c840a)

Hope this helps! ( lmk if i screwed smth up :/ )

**Comment by EngrPips** [comment]
Interesting. You have done a great job, sir.
